.babelrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
We have two config files for Babel:
`config/babel.dev.js` and `config/babel.dev.js`

I suggest to use them instead of `.babelrc`